src/dsp_tools/utils/xml_validation_models.py

```diff
@@ -0,0 +1,143 @@
+from dataclasses import dataclass
+
+import pandas as pd
+from lxml import etree
+
+list_separator = "\n    - "
+medium_separator = "\n----------------------------\n"
+grand_separator = "\n\n---------------------------------------\n\n"
+
+maximum_prints = 50
```

Comment:
I disagree:
- In other files that generate a error string (eg. src/dsp_tools/commands/xmlupload/models/ontology_problem_models.py) These are listed at the top, making it consistent.
- The idea is that the messages will be uniform, meaning other problems should have access to it. I would be happy to look into making these constants for the entire repo, but that should be another PR.
- I intend to add the problem model that generates the error string if the wrong encoding is used with respect to the property in this file. This will also need access to these variables.

Reply:
That sounds like a perfect use case for inheritance: If `Problem` was a class (not a protocol), and the `InconsistentTextValueEncodings` inherits from it, then you can add these fields to the base class, and all children will have the same values.

With the present solution, we have multiple files, each defines its own values, and if we want to change the values, we have to modify multiple files. 

So for right now, you can leave it as it is, but for the future, we should check if inheritance would be a better solution.

Reply:
I added it as a point to the notion page that discusses the config file.
https://www.notion.so/dasch-swiss/New-global-config-object-be2645e1e9a14e5da185dd8b3c6abd8f